Firestore/core/test/unit/local/leveldb_key_test.cc

```diff
@@ -658,9 +665,10 @@ TEST(IndexEntryKeyTest, EncodeDecodeCycle) {
 TEST(IndexEntryKeyTest, Description) {
   AssertExpectedKeyDescription(
       "[index_entries: index_id=1 user_id=user array_value=array "
-      "directional_value=directional document_id=foo-bar?baz!quux]",
+      "directional_value=directional ordered_document_id=<skipped> "
```

Comment:
Why `<skipped>`?

Reply:
It is encoded string, which does not make a lot sense to print.